kubernetes-model-generator/generateModel.sh

```diff
@@ -53,8 +54,40 @@ declare -a modules=(
     "openshift-model-whereabouts"
     "openshift-model-storageversionmigrator"
 )
+
+declare -a extensionModuleParents=(
+    "../extensions/knative/pom.xml"
+    "../extensions/camel-k/pom.xml"
+    "../extensions/camel-k/pom.xml"
+    "../extensions/chaosmesh/pom.xml"
+    "../extensions/service-catalog/pom.xml"
+    "../extensions/tekton/pom.xml"
+    "../extensions/tekton/pom.xml"
+    "../extensions/volumesnapshot/pom.xml"
+)
+
+declare -a extensionModules=(
+    "../extensions/knative/generator"
+    "../extensions/camel-k/generator-v1"
+    "../extensions/camel-k/generator-v1alpha1"
+    "../extensions/chaosmesh/generator"
+    "../extensions/service-catalog/generator"
+    "../extensions/tekton/generator-v1alpha1"
+    "../extensions/tekton/generator-v1beta1"
+    "../extensions/volumesnapshot/generator"
+)
+
 generateAll() {
-  for module in ${modules[*]}
+  echo "Compiling core modules"
+  generateSetOfModules "${modules[@]}"
+  echo "Compiling extensions"
+  extensionInstallCommonModules
```

Comment:
Why isn't this run with the rest of prerequisites (after line `mvn clean install -pl kubernetes-model-common -pl kubernetes-model-jsonschema2pojo`)?